tools/esdlcmd/esdl-publish.cpp

```diff
@@ -79,7 +79,6 @@ class EsdlPublishCmdCommon : public EsdlCmdCommon
                 "   --port <port>                WsESDLConfig service port\n"
                 "   -u, --username <name>        Username for accessing WsESDLConfig service\n"
                 "   -pw, --password <pw>         Password for accessing WsESDLConfig service\n"
-                "   --version <ver>              ESDL service version\n"
```

Comment:
I think we need to keep this line in the base class usage (and update it to "--interface-version,               ESDL interface version\n") since all extensions of this class support should support --version (now --interface-version)
If publish command won't acknowledge this argument, then that class needs to override this usage.